tests/modules/images/webp-uploads/load-tests.php

```diff
@@ -789,6 +790,8 @@ public function it_should_create_webp_for_full_size_which_is_smaller_in_webp_for
 			$this->assertImageNotHasSizeSource( $attachment_id, $size, 'image/webp' );
 		}
 		$this->assertNotSame( $tag, webp_uploads_img_tag_update_mime_type( $tag, 'the_content', $attachment_id ) );
+
+		remove_filter( 'wp_editor_set_quality', array( $this, 'return_webp_image_quality' ), PHP_INT_MAX );
```

Comment:
Please remove. WP's test suite removes filters from test methods automatically. Also, this wouldn't work reliably, if the assertion fails for example.

Reply:
Removed!